FINS/FINS-IOC-01App/Db/sans2d-vac-status.template

```diff
@@ -7,28 +7,53 @@
 
 record(mbbi, "$(FULL_PREFIX)$(SYSTEM):STATUS") {
     field(DESC, "$(SYSTEM) Status")
-    field(DTYP, "asynInt32")
-    field(INP,  "@asyn(PLC, $(ADDR), 5.0) FINS_DM_READ")
-    field(SCAN, "1 second")
-    field(NOBT, 4)
+    field(INP, "$(FULL_PREFIX)$(SYSTEM):STATUS:_SET_UNKNOWN CP MS")
     
+    field(ZRVL, "0")
     field(ONVL, "1")
     field(TWVL, "2")
     field(FRVL, "4")
     field(EIVL, "8")
     
+    field(ZRST, "ERROR: STATUS UNKNOWN")
     field(ONST, "ATMOSPHERE")
     field(TWST, "VAC DOWN")
     field(FRST, "AT VACUUM")
     field(EIST, "VENTING")
-    
-    field(SIML, "$(P)$(Q)SIM")
-    field(SIOL, "$(P)$(Q)SIM:$(SYSTEM):STATUS")
+
+    field(ZRSV, "MINOR")
 
     info(archive, "VAL")
     info(INTEREST, "HIGH")
 }
 
+record(longin, "$(FULL_PREFIX)$(SYSTEM):STATUS:_RAW") {
```

Comment:
Is there any reason this record exists and why STATUS:_SET does not read directly from hardware?

Reply:
Because we cannot simulate an mbbi asyn record in simulation in order to test it.